packages/core/src/MultiButton/stories/MultiButton.stories.tsx

```diff
@@ -288,6 +288,17 @@ export const Test: StoryObj = {
             ))}
           </HvMultiButton>
         </div>
+        <div className="grid gap-xs">
+          <HvMultiButton>
+            <HvButton>Label 1</HvButton>
+          </HvMultiButton>
+          <HvMultiButton>
+            <HvButton selected>Label 1</HvButton>
+          </HvMultiButton>
+          <HvMultiButton>
+            <HvButton disabled>Label 1</HvButton>
+          </HvMultiButton>
+        </div>
```

Comment:
this is dragging all the other samples unnecessarily. let's inline this alongside the `width: fit-content` icon-only `HvMultiButton`

```tsx
<div className="flex flex-1 [&>*]:flex-1 gap-xs">
  <HvMultiButton>
    ...
```

![image](https://github.com/user-attachments/assets/1c69c1dd-334c-4f96-8c51-415057c41e55)